src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -2343,6 +2344,16 @@ public boolean hasResourcePack() {
         return resourcePackStatus == PlayerResourcePackStatusEvent.Status.SUCCESSFULLY_LOADED;
     }
 
+    @Override
+    public PlayerProfile getPlayerProfile() {
+        return getProfile();
+    }
+
+    @Override
+    public void setPlayerProfile(PlayerProfile playerProfile) {
+        throw new UnsupportedOperationException("Not implemented yet.");
```

Comment:
This method should ensure that only GameProfiles with the same UUID can be set, Else throw an exception.
Modification of the players UUID shouldn’t be allowed after creation.

Reply:
Again unrelated to this PR (since this is just a stub), but it is in fact designed to make it easy to change a player's identity. It is specifically designed to make it possible to change the identity of the player, or something like that. (Not entirely sure, see https://ci.destroystokyo.com/userContent/apidocs/org/bukkit/entity/Player.html#setPlayerProfile-com.destroystokyo.paper.profile.PlayerProfile-)